### PR TITLE
fixing resource file writing.

### PR DESCRIPTION
**Pull request by srussellextensis**
Addresses https://github.com/uliwitness/ReClassicfication/issues/1

### CODE REVIEW

File: InterfaceLib/FakeResources.c [hunk 1]
```diff
@@ -450,9 +452,9 @@ void	FakeUpdateResFile( int16_t inFileRefNum )
 			resMapOffset += 4;
 			fwrite( *currMap->typeList[x].resourceList[y].resourceHandle, 1, theSize, currMap->fileDescriptor );
 			resMapOffset += theSize;
-			
-			refListSize += currMap->typeList[x].numberOfResourcesOfType * (2 + 2 + 1 + 3 + 4);
 		}
+
+		refListSize += currMap->typeList[x].numberOfResourcesOfType * (2 + 2 + 1 + 3 + 4);
```
Comment (MichaelBuckley) [inline InterfaceLib/FakeResources.c]: I know all the magic numbers are from the original code, but any chance we could get consts for these ?

File: InterfaceLib/FakeResources.c [hunk 2]
```diff
@@ -465,22 +467,25 @@ void	FakeUpdateResFile( int16_t inFileRefNum )
 	uint32_t		resMapLength = 0;
 	FakeFSeek( currMap->fileDescriptor, resMapOffset, SEEK_SET );
 	
-	FakeFSeek( currMap->fileDescriptor, 4 + 4 + 4 + 4 + 4 + 2, SEEK_CUR );
-	resMapLength += 4 + 4 + 4 + 4 + 4 + 2;
+	resMapLength += (4 + 4 + 4 + 4) + 4 + 2;	// reserved: copy of resource header, next resource handle, file ref
```
Comment (MichaelBuckley) [inline InterfaceLib/FakeResources.c]: Do the parentheses add anything here?

Comment (srussellextensis) [reply InterfaceLib/FakeResources.c]: was just trying to group them as the copy of the resource header

File: InterfaceLib/FakeResources.c (hunk 1 above)
Comment (srussellextensis) [reply InterfaceLib/FakeResources.c]: I'm not sure I'd want to do it for every little size, but I sort of considered it for repeating clusters like this. I can do that for sure, and I'll think about the former.

**Review by MichaelBuckley — APPROVED** [review]

**Pull request merged**